tutorials/introductory/usage.py

```diff
@@ -130,22 +130,23 @@
 #
 # Coding styles
 # =============
-
-# The object-oriented and the pyplot interfaces
-# ---------------------------------------------
+#
+# The explicit and the implicit approach of programming
+# -----------------------------------------------------
 #
 # As noted above, there are essentially two ways to use Matplotlib:
 #
-# - Explicitly create figures and axes, and call methods on them (the
-#   "object-oriented (OO) style").
-# - Rely on pyplot to automatically create and manage the figures and axes, and
-#   use pyplot functions for plotting.
+# - Explicitly create Figures and Axes, and call methods on them (the explicit
+#   or "object oriented programming (OOP) style").
+# - Rely on pyplot to automatically create and manage the Figures and Axes, and
+#   use pyplot functions for plotting (the implicit style).
 #
-# So one can use the OO-style
+# So one can use the explicit style
```

Comment:
Maybe its just me, but I have a mental block with words that are almost the same but mean opposite things; for explicit/implicit I have to _every_ time remember which is which.  I'd prefer using "OO" and "pyplot" here, and I think that is what has been historically done.

Reply:
Frankly speaking I'm rather neutral/indifferent here, just following the [style guide](https://matplotlib.org/stable/devel/style_guide.html#terminology) that discourages the use of "OO-style" (it even calls it "incorrect")

Reply:
reverted the OO/pyplot/explicit/implicit changes